### PR TITLE
Migguide structure

### PR DESCRIPTION
**Pull request by rgstephens**
Migration guide file structure updates per #3947

### CODE REVIEW

**Comment by akelad** [comment]
Thanks a lot for submitting this PR. Before we review it, could you please remove the `.history` files?

File: docs/migration-guide.rst [hunk 1]
```diff
@@ -41,6 +41,10 @@ General
   <https://legacy-docs.rasa.com/docs/nlu/0.15.1/migrations/?&_ga=2.218966814.608734414.1560704810-314462423.1543594887#id1>`.
   These two files should be merged into a single file either named ``config.yml``, or passed via the ``--config`` parameter.
 
+- While not required, it's recommended that your project filenames and directory structure follow the
+  convention created by the `rasa init` command  (see :ref:`command-line-interface <create-a-new-project>`).
```
Comment (erohmensing) [inline docs/migration-guide.rst]:
```suggestion
  convention created by the ``rasa init`` command  (see :ref:`command-line-interface`).
```
In `.rst` files, the double `` has the same functionality as single ones in e.g. github. Single ones will italicize in rst

**Comment by erohmensing** [comment]
@rgstephens still interested in merging these changes? I'd commit the change myself but I can't push to your fork.

**Comment by erohmensing** [comment]
With no response, I will close this. Let me know if you'd like to re-open and fix the one suggestion to merge.

**Pull request closed without merging**